__tests__/components/forecast-summaries.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummary from '../../src/components/forecast-summary';
+import ForecastSummaries from '../../src/components/forecast-summaries';
+
+describe('Forecast Summaries', () => {
+  const forecasts = [
+    {
+      date: 123,
+      description: 'date1',
+      icon: 'icon1',
+      temperature: {
+        max: 999,
+      },
+    },
+
+    {
+      date: 456,
+      description: 'date2',
+      icon: 'icon2',
+      temperature: {
+        max: 777,
+      },
+    },
+  ];
+
```

Comment:
Remember that you can user `beforeEach()` blocks to handle the setup for each test.